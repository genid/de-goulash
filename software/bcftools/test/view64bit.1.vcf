##fileformat=VCFv4.2
##INFO=<ID=MPOS,Number=A,Type=Integer,Description="dummy">
##INFO=<ID=XPOS,Number=.,Type=Integer,Description="dummy">
##INFO=<ID=NALOD,Number=A,Type=Float,Description="dummy">
##INFO=<ID=NLOD,Number=A,Type=Float,Description="dummy">
##INFO=<ID=POPAF,Number=A,Type=Float,Description="dummy">
##contig=<ID=chr1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	1	.	G	C	.	.	MPOS=-2147483641;XPOS=-2147483641,-2147483641,-2147483641;NALOD=-8.279e-01;NLOD=15.45;POPAF=6.00
chr1	2	.	G	C	.	.	MPOS=-2147483648;XPOS=-2147483648,-2147483648,-2147483648;NALOD=-8.279e-01;NLOD=15.45;POPAF=6.00
