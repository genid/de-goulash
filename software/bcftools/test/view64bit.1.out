chr1	1	.	G	C	.	.	MPOS=.;XPOS=.,.,.;NALOD=-0.8279;NLOD=15.45;POPAF=6
chr1	2	.	G	C	.	.	MPOS=.;XPOS=.,.,.;NALOD=-0.8279;NLOD=15.45;POPAF=6
